src/main/scala/io/shiftleft/fuzzyc2cpg/FuzzyC2Cpg.scala

```diff
@@ -175,7 +175,7 @@ class FuzzyC2Cpg(outputModuleFactory: CpgOutputModuleFactory) {
       case ex: RuntimeException => {
         logger.warn("Cannot parse module: " + filename + ", skipping")
         logger.warn("Complete exception: ", ex)
-        return
+        throw ex
```

Comment:
Does anybody catch this now? The idea here was to proceed with the next module - even if one of the modules fails to parse correctly. That way, failing on one file doesn't mean that no CPG can be created.